### ISSUE TITLE
cacert.pem

### ISSUE THREAD

**Issue by a8nJ** [bagder/ca-bundle#8]
These two appear to be the exact same file:

- https://curl.haxx.se/ca/cacert.pem
- https://github.com/bagder/ca-bundle/blob/master/ca-bundle.crt

The issue is that some programs (Git), are expecting this path:

~~~
/usr/ssl/certs/ca-bundle.crt
~~~

So to people coming across the `cacert.pem` file, they might think its the wrong
file, or that they need to convert the file, when in reality all they need to do
is rename the file.

It might be helpful if the file had the same name in both places. Or, failing
that, it would be helpful for an explanation in one (or both) places that these
two are exactly the same file with different names.

**Comment by a8nJ** [comment]
Just to add some more evidence, a quick search shows that these two names have
caused people confusion again, and again, and again:

- https://serverfault.com/questions/455486
- https://serverfault.com/questions/62496
- https://stackoverflow.com/questions/28858351
- https://stackoverflow.com/questions/3160909
- https://stackoverflow.com/questions/34087962
- https://stackoverflow.com/questions/39356413
- https://stackoverflow.com/questions/58725457
- https://superuser.com/questions/442793

**Comment by bagder** [comment]
> These two appear to be the exact same file:

This github repository explains in several places that it is a *backup service* for the one on the curl website. It would be a rather poor backup if it didn't provide the exact same file. It will just possibly be a bit delayed compared to the service on the curl site.

> It might be helpful if the file had the same name in both places

I very much doubt that helps. If people can't realize what the file is for and they get stuck on the name, then I would have to provide the file with the exact right name(s) for every such use case. And programs and systems are free to call it whatever they like. In many cases I provided and offered the file on the curl site long before those other programs were even born. Also, I can't *change* the name on the curl site now as there are lots of automated systems downloading it. I could possibly add aliases for it, but again I'm not at all convinced that actually benefits the human race.

I've added a mention on the curl site about the fact the file is often used under other names and spell out two examples.
